02-Reproject-Raster-In-R.Rmd

```diff
@@ -231,12 +244,11 @@ res(DTM_hill_UTMZ18N_HARV)
 Once we have reprojected the raster, we can try to plot again!
 
 ```{r plot-projected-raster }
-
 #plot newly reprojected hillshade
 plot(DTM_hill_UTMZ18N_HARV,
     col=grey(1:100/100),
     legend=F,
-    main="NEON Hillshade - DTM\n Harvard Forest")
+    main="NEON DTM with Hillshade\n Harvard Forest")
```

Comment:
I'm noticing now that our plot titles are inconsistent.

NEON DTM with Hillshade\nHarvard Forest Field Site